Doc/library/winreg.rst

```diff
@@ -691,64 +692,82 @@ For more information, see `Registry Value Types
 .. data:: REG_BINARY
 
    Binary data in any form.
+   *value* must be a :term:`bytes-like object` in Python for this type.
```

Comment:
Specify not only accepted types, but also returned types. For example, REG_BINARY accepts a bytes-like object and None but returns a bytes object or None for empty value. Most of other types accept None but do not return it.